.TH "Issue1340Test" 3 "Tue Apr 14 2015" "Version 1.0" "VirtualSCADA" \" -*- nroff -*-
.ad l
.nh
.SH NAME
Issue1340Test \- 
.SH SYNOPSIS
.br
.PP
.PP
Inherits \fBPHPUnit_Framework_TestCase\fP\&.
.SS "Public Member Functions"

.in +1c
.ti -1c
.RI "\fBtestLargeStderrOutputDoesNotBlock\fP ()"
.br
.ti -1c
.RI "\fBtestLargeStderrOutputDoesNotBlockInIsolation\fP ()"
.br
.ti -1c
.RI "\fBtestPhpNoticeIsCaught\fP ()"
.br
.ti -1c
.RI "\fBtestPhpNoticeWithStderrOutputIsAnError\fP ()"
.br
.ti -1c
.RI "\fBtestFatalErrorDoesNotPass\fP ()"
.br
.in -1c
.SS "Static Public Member Functions"

.in +1c
.ti -1c
.RI "static \fBonShutdown\fP ()"
.br
.in -1c
.SS "Additional Inherited Members"
.SH "Detailed Description"
.PP 

.PP
\fBSee also:\fP
.RS 4
https://bugs.php.net/bug.php?id=51800 
.RE
.PP

.PP
Definition at line 5 of file Issue1340Test\&.php\&.
.SH "Member Function Documentation"
.PP 
.SS "static onShutdown ()\fC [static]\fP"

.PP
Definition at line 65 of file Issue1340Test\&.php\&.
.SS "testFatalErrorDoesNotPass ()"

.PP
Definition at line 58 of file Issue1340Test\&.php\&.
.SS "testLargeStderrOutputDoesNotBlock ()"
Also fails despite no isolation, because a phpt test is executed in subprocess on its own\&. 
.PP
Definition at line 16 of file Issue1340Test\&.php\&.
.SS "testLargeStderrOutputDoesNotBlockInIsolation ()"

.PP
Definition at line 28 of file Issue1340Test\&.php\&.
.SS "testPhpNoticeIsCaught ()"
Undefined variable: foo 
.PP
Definition at line 39 of file Issue1340Test\&.php\&.
.SS "testPhpNoticeWithStderrOutputIsAnError ()"
Undefined variable: foo 
.PP
Definition at line 49 of file Issue1340Test\&.php\&.

.SH "Author"
.PP 
Generated automatically by Doxygen for VirtualSCADA from the source code\&.
